runtime/gc_structs/ConstantPoolObjectSlotIterator.cpp

```diff
@@ -35,18 +35,22 @@
 
 
 /**
+ * If condyOnly flag is not set:
  * @return the next object slot in the constant pool
  * @return NULL if there are no more object references
+ *
+ * If condyOnly flag is set and Java version is SE11 and above:
+ * @return the next Condy reference from the constant pool
+ * @return NULL if there are no more references
  */
 j9object_t *
-GC_ConstantPoolObjectSlotIterator::nextSlot()
-{
+GC_ConstantPoolObjectSlotIterator::nextSlot(MM_EnvironmentBase *env, Boolean condyOnly) {
```

Comment:
condyOnly should be a member of the class, passed only once to the constructor. no need to pass it every time on nextSlot()